### PR TITLE
Remove unsupported release.toml setting

### PR DESCRIPTION
**Pull request by rfuest**
Remove unsupported release.toml setting

### CODE REVIEW

File: release.toml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-no-dev-version = true
```
Comment (jamwaffles) [inline release.toml:1]: GH won't let me add a suggestion but I've switched this to `dev-version = false` in other projects, which retains the current behaviour.

**Review by jamwaffles — CHANGES_REQUESTED** [review]

**Review by jamwaffles — APPROVED** [review]
Ah ignore me, I didn't know the option was removed entirely upstream.

**Comment by rfuest** [comment]
> Ah ignore me, I didn't know the option was removed entirely upstream.

This did also confuse me when I searched the `cargo-release` changelog for a replacement and didn't notice that the replacement had also be removed.

**Comment by jamwaffles** [comment]
I've just released new versions of both crates. I'm not sure why the permissions won't let you release yourself. I haven't changed anything in a long time, so maybe Github updated something on their side.

**Comment by rfuest** [comment]
Thanks. I'm not 100% sure if it would have failed, perhaps I messed something else up and got the git error message because of that.

**Pull request merged**